cdap-app-fabric/src/main/java/org/apache/twill/yarn/YarnSecureStore.java

```diff
@@ -17,26 +17,44 @@
  */
 package org.apache.twill.yarn;
 
+import com.google.common.base.Throwables;
 import org.apache.hadoop.security.Credentials;
+import org.apache.hadoop.security.UserGroupInformation;
 import org.apache.twill.api.SecureStore;
 
+import java.io.IOException;
+
 /**
  * A {@link SecureStore} for hadoop credentials.
  */
 public final class YarnSecureStore implements SecureStore {
```

Comment:
hmmm ok so from my earlier comment, this class doesn't have ``update()``. Seems weird that we have to inject implementations though, as oppsed to always injecting SecureStore and letting Guice do bindings.